### ISSUE TITLE
Routine Pulumi update drops Python descriptions

### ISSUE THREAD

**Issue by t0yv0** [pulumi/pulumi-azure-native#2736]
### What happened?

Would like to land https://github.com/pulumi/pulumi-azure-native/pull/2735

However  it's full of dropped Python docstrings, I suspect this is a blocker bug? 

```
    """
    Status for individual validator after running diagnostics.
    """
    NONE = "None"
    RUNNING = "Running"
    OK = "OK"
```

### Expected Behavior

No documentation dropped.

### Steps to reproduce

See PR.

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
@t0yv0 It sounds like it must be a pu/pu issue? Should I move the issue there?

**Comment by t0yv0** [comment]
Yeah I wanted to narrow down the repro and pass along, looking into it shortly.

**Comment by t0yv0** [comment]
Yes I was able to narrow it down enough so we can exclusively attribute it to Pulumi CLI update. I opened https://github.com/pulumi/pulumi/issues/13921 in platform with detailed repro.

**Comment by mikhailshilkov** [comment]
The upstream issue has been fixed and Azure Native is now on the latest pu/pu
